packages/datatrak-web/src/layout/UserMenu/UserInfo.tsx

```diff
@@ -75,27 +48,29 @@ const UserName = styled(Typography)`
   }
 `;
 
-const AuthButtons = styled.div`
+const AuthButtons = styled(Box)`
   ${({ theme }) => theme.breakpoints.down('sm')} {
     display: none;
   }
 `;
+
 /**
  * This is the displayed user name OR the login/register buttons on desktop
  */
-export const UserInfo = ({ openProjectModal }: { openProjectModal: () => void }) => {
-  const user = useCurrentUser();
+export const UserInfo = () => {
+  const { isLoggedIn, projectId, userName } = useCurrentUser();
+
+  const StyledProjectButton = styled(ChangeProjectButton)`
```

Comment:
I probably wouldn't put a styled component inside of another component, not sure if there are performance impacts with re-rendering, but it also just doesn't match our pattern - probably best to move it outside.

Reply:
This causes an error, but I’m not entirely sure why. Have you seen this before?

```
Uncaught ReferenceError: Cannot access 'ChangeProjectButton' before initialisation at UserInfo.tsx(🥥)
```

where 🥥 is this very line (`const StyledProjectButton = ...`).

I guess React tries to initialise `UserInfo` before it’s created `ChangeProjectButton`?

Reply:
Did a bit of a refactor which:

- consolidates **all** styling of this component into a single place, obviating the need for this styled component in the first place,
- nullifies the weird ReferenceError.